moesifmiddleware.go

```diff
@@ -75,6 +88,33 @@
 			 multiWriter,
 		 )
 
+		// Disable TransactionId by default
+		disableTransactionId := false
+		// Try to fetch the disableTransactionId from the option
+		if isEnabled, found := moesifOption["disableTransactionId"].(bool); found {
+			disableTransactionId = isEnabled
+		}
+
+		// Add transactionId to the headers
+		if !disableTransactionId {
+			// Try to fetch the transactionId from the header
+			reqTransId := request.Header.Get("X-Moesif-Transaction-Id")
+			// Set the transationId
+			transationId := reqTransId
+			// Check if need to generate transactionId
+			if len(transationId) == 0 {
```

Comment:
Is it possible for `transactionId` to be `nil`?

Reply:
yes, it is possible for transactionId to be nil if the user has set the flag